pymeasure/experiment/parameters.py

```diff
@@ -46,6 +46,11 @@ class Parameter:
 
     def __init__(self, name, default=None, ui_class=None, group_by=None, group_condition=True):
         self.name = name
+        separator = ": "
+        if separator in name:
+            raise ValueError(f"The provided name argument '{name}' contains the "
+                             "separator '{separator}'.")
```

Comment:
Because the string is broken in two, you need an additional `f` in front of the second part of the string:

```suggestion
                             f"separator '{separator}'.")
```